BlackNectar/BlackNectar/StoresDataStruct.swift

```diff
@@ -34,10 +35,20 @@ extension Store {
     
     init?(json: NSDictionary) {
         
-        guard let storeName = json["store_name"] as? String,
-            let addressJSON = json["address"] as? NSDictionary,
-            let storeImageString = json["main_image_url"] as? String,
-            let storeImageURL = URL(string: storeImageString)
+        let storeImageString = json ["main_image_url"] as? String
+        
+        if storeImageString == nil {
+
+            storeImageURL = URL(string:"https://s3.amazonaws.com/ODNUploads/5404f487678f1532dfdbcc6479placeholder_food_item_2.png")
```

Comment:
It is better to have no image at all, and just a `UIColor` instead.

So, whenever the cell is loaded
```swift
if let url = store.imageURL {
setImage(url)
}
else {
cell.image.image = nil
cell.image.backgroundColor = Colors.PRIMARY
}
```